src/dropdown.js

```diff
@@ -33,6 +35,18 @@ export default class extends Controller {
       !this.menuTarget.classList.contains(this.toggleClass)
     ) {
       this.menuTarget.classList.add(this.toggleClass)
+      this._activeClassList.forEach((klass) =>  this.activeTarget.classList.remove(klass))
     }
   }
+
+  get activeTarget() {
+    return this.data.has('activeTarget') ? document.querySelector(this.data.get('activeTarget')) : this.element
+  }
+
+  get _activeClassList() {
+    if (!this.activeClass) return [];
+    const classList = this.activeClass.split(' ')
+    return classList
```

Comment:
Does refactoring to this work?

```javascript
get _activeClassList() {
  return (!this.activeClass ? [] : this.activeClass.split(' '))
}
```

Reply:
Yes it does. I'll push this up now.

Reply:
🙏

Reply:
Pushed. I just decided to squash down the commits as well.